packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -306,7 +306,7 @@ export class ModelInstanceDocument<T = Record<string, any>> extends Stream {
         await _ensureAuthenticated(signer)
         // When did has a parent, it has a capability, and the did issuer (parent) of the capability
         // is the stream controller
-        controller = signer.did.hasParent ? signer.did.parent : signer.did.id
+        controller = signer.did.did.hasParent ? signer.did.did.parent : signer.did.did.id
```

Comment:
I'm not a big fan of this `did.did` thing.  Can we make the names clearer or in same way structure this so that it's clear what the difference between these two objects is?

Reply:
Yeah, we can expose id, parent, and hasParent in ThreadedDid.